redbot/cogs/warnings/warnings.py

```diff
@@ -1,6 +1,7 @@
 from collections import namedtuple
 from typing import Union, Optional
 
+import contextlib
```

Comment:
Note really a necesity but...

```
from collections import namedtuple
from typing import Union, Optional

import contextlib
import discord
```
should be 
```
import contextlib
from collections import namedtuple
from typing import Union, Optional

import discord
```

This one is just a nitpick for import organization 😈